src/app/components/Account.tsx

```diff
@@ -1,30 +1,56 @@
 import 'index.css'
+import * as React from 'react'
+import classnames from 'classnames'
+import { useNavigate } from 'react-router-dom'
 
 import { useAuth } from 'app/hooks/useAuth'
-
 import SvgAccount from './SVG/SvgAccount'
 
 export default function Account() {
   const { user } = useAuth()
+  const navigate = useNavigate()
+  const [popup, setPopup] = React.useState(false)
+  const popupStyle = classnames(styles.popup, { hidden: !popup })
 
   const avatar =
     'https://upload.wikimedia.org/wikipedia/commons/thumb/4/49/Flag_of_Ukraine.svg/375px-Flag_of_Ukraine.svg.png'
 
+  const handleOnCLickAccount = () => {
+    setPopup(!popup)
+  }
+  const handleOnCLickLogout = () => {
+    localStorage.removeItem('jwtAccessToken')
+    localStorage.removeItem('jwtRefreshToken')
```

Comment:
Instead of direct implementing logic of logout in `Account` component, let's implement it in `useAuth`